telescope/ERQTelescopePID.cxx

```diff
@@ -92,53 +92,39 @@ InitStatus ERQTelescopePID::Init() {
 //--------------------------------------------------------------------------------------------------
 void ERQTelescopePID::Exec(Option_t* opt) { 
   Reset();
-  
-  for (const auto itTrackBranches : fQTelescopeTrack) {
-
-    LOG(DEBUG) << " Work with traks in " << itTrackBranches.first << FairLogger::endl;
-
-    for (Int_t iTrack(0); iTrack < itTrackBranches.second->GetEntriesFast(); iTrack++){
-
-      ERQTelescopeTrack* track = (ERQTelescopeTrack*)itTrackBranches.second->At(iTrack);
-
-      for (const auto itParticesBranches : fQTelescopeParticle[itTrackBranches.first]){
-
-        Int_t pdg = itParticesBranches.first;
-
-        //sum edep on road from telescope hit produce station to CsI and edep in CsI digi
-        Double_t csIedep = FindCsIEdepByTrack(track,pdg);
```

Comment:
In a common case, we lose CsI consideration. It should be returned to the PID procedure

Reply:
Or FindDigiEdepByNode(...) may become more general